platform/default/src/mbgl/layermanager/layer_manager.cpp

```diff
@@ -32,16 +32,56 @@ class LayerManagerDefault final : public LayerManager {
 };
 
 LayerManagerDefault::LayerManagerDefault() {
+#if defined(MBGL_LAYER_FILL_DISABLE_RUNTIME)
```

Comment:
`LayerManagerDefault` does not have the `addLayerTypeCoreOnly()` method, here we should just check for `MBGL_LAYER_ ... _DISABLE_ALL` flag.